game_eggs/steamcmd_servers/the_isle/evrima/egg-the-isle.json

```diff
@@ -15,14 +15,14 @@
     "file_denylist": [],
     "startup": "\/home\/container\/steamcmd\/steamcmd.sh +force_install_dir \/home\/container +login Anonymous +app_update 412680 -beta evrima +quit && \/home\/container\/TheIsle\/Binaries\/Linux\/TheIsleServer-Linux-Shipping -Port=$SERVER_PORT -ini:Engine:[EpicOnlineServices]:DedicatedServerClientId=xyza7891gk5PRo3J7G9puCJGFJjmEguW -ini:Engine:[EpicOnlineServices]:DedicatedServerClientSecret=pKWl6t5i9NJK8gTpVlAxzENZ65P8hYzodV8Dqe5Rlc8",
     "config": {
-        "files": "{\r\n    \"\/TheIsle\/Saved\/Config\/LinuxServer\/Game.ini\": {\r\n        \"parser\": \"file\",\r\n        \"find\": {\r\n            \"ServerName\": \"ServerName={{server.build.env.SERVER_NAME}}\",\r\n            \"MaxPlayerCount\": \"MaxPlayerCount={{server.build.env.PLAYER_COUNT}}\",\r\n            \"MaxPlayers\": \"MaxPlayers={{server.build.env.PLAYER_COUNT}}\",\r\n            \"RconEnabled\": \"RconEnabled={{server.build.env.RCON}}\",\r\n            \"RconPort\": \"RconPort={{server.build.env.RCON_PORT}}\",\r\n            \"RconPassword\": \"RconPassword={{server.build.env.RCON_PASSWORD}}\",\r\n            \"QueuePort\": \"QueuePort={{server.build.env.QUEUE_PORT}}\",\r\n            \"discord\": \"discord={{server.build.env.DISCORD}}\"\r\n        }\r\n    }\r\n}",
+        "files": "{\r\n    \"\/TheIsle\/Saved\/Config\/LinuxServer\/Game.ini\": {\r\n        \"parser\": \"file\",\r\n        \"find\": {\r\n            \"ServerName\": \"ServerName={{server.build.env.SERVER_NAME}}\",\r\n            \"MaxPlayerCount\": \"MaxPlayerCount={{server.build.env.PLAYER_COUNT}}\",\r\n            \"MaxPlayers\": \"MaxPlayers={{server.build.env.PLAYER_COUNT}}\",\r\n            \"RconEnabled\": \"RconEnabled{{server.build.env.RCON}}\",\r\n            \"RconPort\": \"RconPort={{server.build.env.RCON_PORT}}\",\r\n            \"RconPassword\": \"RconPassword={{server.build.env.RCON_PASSWORD}}\",\r\n            \"QueuePort\": \"QueuePort={{server.build.env.QUEUE_PORT}}\",\r\n            \"discord\": \"discord={{server.build.env.DISCORD}}\",\r\n            \"bEnableHumans\": \"bEnableHumans{{server.build.env.HUMANS}}\"\r\n        }\r\n    }\r\n}",
```

Comment:
No need to remove the = from RconEnabled here and the same for bEnableHumans

Reply:
The reason that I ended up doing it this way is because if I only use "in:true,false", I get an error everytime I try to use it. 
![image](https://user-images.githubusercontent.com/54645039/223051302-7ea4e109-6abe-4e16-a995-7cb59e510a94.png)

Reply:
> The reason that I ended up doing it this way is because if I only use "in:true,false", I get an error everytime I try to use it. ![image](https://user-images.githubusercontent.com/54645039/223051302-7ea4e109-6abe-4e16-a995-7cb59e510a94.png)

What is the regex you set to that variable.

Reply:
This is the error I'm getting in the console. 
```
{"errors":[{"code":"ValidationException","status":"422","detail":"The selected value is invalid.","meta":
{"source_field":"value","rule":"in"}}]}
```
I'm simply trying to use `required|in:false,true` as the input, but whenever I do that it changes to the button and doesn't seem to work. If i do `boolean` then it works, but then the value in the file is either 0 or 1 and that's no good.

Reply:
it should work

Reply:
it works if you force it to be a string, I cleaned the egg a little up `required|string|in:false,true`